src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -313,6 +314,22 @@ void SX1280Driver::SetPacketParamsFLRC(uint8_t HeaderType,
     buf[1] = (uint8_t)(syncWord >> 16);
     buf[2] = (uint8_t)(syncWord >> 8);
     buf[3] = (uint8_t)syncWord;
+
+    // DS_SX1280-1_V3.2.pdf - 16.4 FLRC Modem: Increased PER in FLRC Packets with Synch Word
+    if ((cr == SX1280_FLRC_CR_1_2) && ((buf[0] == 0x8C && buf[1] == 0x38) || (buf[0] == 0x63 && buf[1] == 0x0E)))
+    {
+        uint8_t temp = buf[0];
+        buf[0] = buf[1];
+        buf[1] = temp;
+    }
+    else
+    {
+        // Not done for unused coding rates to save space.
+        DBGLN("!!! WARNING !!!");
+        DBGLN("Check datasheet errata for forbidden sync word bytes when using coding rates 3/4 and 1/1.");
+        DBGLN("!!! WARNING !!!");
```

Comment:
Personally I'd make these ERRLN, and then maybe even stick a `while(1);` after it so it hangs!

Reply:
That's sounds much better.  Will do.